app/assets/v2/js/pages/bounty_details.js

```diff
@@ -65,6 +65,15 @@ const token_value_to_display = function(val) {
   return Math.round((parseInt(val) / Math.pow(10, document.decimals)) * 1000) / 1000;
 };
 
+const case_insensitive_cmp = function(val1, val2) {
+  if (val1 && val2) {
+    if (typeof val1 === 'string' && typeof val2 === 'string') {
```

Comment:
Can we mage this into 1 if statement + make it into an arrow functions ?

Reply:
Thanks for your review @thelostone-mc. I add a new commit for it. Should I squash them?